masterlist.yaml

```diff
@@ -24105,6 +24105,13 @@ plugins:
     inc: [ 'Paper World Map Unofficial Fix.esp' ]
     req: [ *FlatMapMarkers ]
 
+  - name: 'FWMF for Paper Map by Mirhayasu.esp'
+    group: 'Late Loaders'
+    after:
+      - 'DynDOLOD.esp'
```

Comment:
Both of those rules together are just going to cause a bunch of sorting errors.
As I've mentioned in #2335 

> Other paper map mods are being placed in the Worldspace Settings group, so it should be fine there as well.
Even with that grouping or placing it last for that matter, it may still require patching with other mod changing the Worldspace records. Things like map markers, lighting overhauls, water mods, paper maps will modify Worldspace records.

Generally DynDOLOD should be the last thing you do for your load order.
It will carry forward the last value in each record it copies, so it makes no sense to load other files after it.
If your map isn't working, you either have a conflict or your dynamic patches (Bashed patch, DynDOLOD etc., need to be regenerated.
If `FWMF for Paper Map by Mirhayasu.esp` is placed in the Worldspace Settings group then there shouldn't be much that can overwrite it besides other paper maps, map markers and worldspace tweaks.

Reply:
```
- name: 'FWMF for Paper Map by Mirhayasu.esp'
    group: 'Worldspace Settings'
    url:
      - 'https://www.nexusmods.com/skyrimspecialedition/mods/29932'
```
 Would this work as intended